aicsimageio/readers/array_like_reader.py

```diff
@@ -360,6 +361,7 @@ def __init__(
                         attrs={constants.METADATA_UNPROCESSED: None},
                     )
                 )
+        self._physical_pixel_sizes = physical_pixel_sizes
```

Comment:
A part of me says: "The use should simply provide the `PhysicalPixelSizes` object instead. Would shorten the typing by quite a lot.

Reply:
Or if you don't want to do that, at least just allow it? Also maybe do the casting to `PhysicalPixelSizes` here in the `__init__` instead of in the prop access.

Reply:
Totally agree that the conversion to the PhysicalPixelSizes object could happen here first, and be stored internally as a PhysicalPixelSizes object.
Then the `physical_pixel_sizes` method can just be a one-liner...